crates/boundless-cli/src/bin/boundless-cli.rs

```diff
@@ -252,6 +253,12 @@ struct SubmitOfferRequirements {
     /// Journal prefix to use as the predicate in the requirements.
     #[clap(long)]
     journal_prefix: Option<String>,
+    /// Address of the callback to use in the requirements.
+    #[clap(long)]
+    callback_addr: Option<Address>,
+    /// Gas limit of the callback to use in the requirements.
+    #[clap(long)]
+    callback_gas_limit: Option<u64>,
```

Comment:
Adding `requires` options will prevent someone from specifying one, but forgetting the other. In the code below, that would result in no callback being added, which they may not figure out right away (bad UX).
```suggestion
    /// Address of the callback to use in the requirements.
    #[clap(long, requires = "callback_gas_limit")]
    callback_addr: Option<Address>,
    /// Gas limit of the callback to use in the requirements.
    #[clap(long, requires = "callback_addr")]
    callback_gas_limit: Option<u64>,
```